tests/kafkatest/tests/client/client_compatibility_features_test.py

```diff
@@ -23,18 +23,26 @@
 from kafkatest.services.zookeeper import ZookeeperService
 from kafkatest.services.kafka import KafkaService
 from ducktape.tests.test import Test
-from kafkatest.version import DEV_BRANCH, LATEST_0_10_0, LATEST_0_10_1, LATEST_0_10_2, V_0_10_1_0, KafkaVersion
+from kafkatest.version import DEV_BRANCH, LATEST_0_10_0, LATEST_0_10_1, LATEST_0_10_2, V_0_11_0_0, V_0_10_1_0, KafkaVersion
 
 def get_broker_features(broker_version):
     features = {}
     if (broker_version < V_0_10_1_0):
+        features["create-topics-supported"] = False
+    else:
+        features["create-topics-supported"] = True
```

Comment:
Why are these in a separate block when compared to `offsets-for-times-supported`, etc.